ui/src/app/join-lobby-link/join-lobby-link.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit } from '@angular/core';
+import { ApiService } from '../_services/api.service';
+import { ToastrService } from 'ngx-toastr';
+import { RedirectionService } from '../_services/redirection.service';
+import { ActivatedRoute } from '@angular/router';
+import { FormBuilder, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-join-lobby-link',
+  templateUrl: './join-lobby-link.component.html',
+  styleUrls: ['./join-lobby-link.component.scss']
+})
+export class JoinLobbyLinkComponent implements OnInit {
+
+  lobbyId;
+
+  joinLobbyForm = this.fb.group({ userName: ['', Validators.required ],});
```

Comment:
I would be better to use a single `FormControl` here instead of a `FormGroup`.

You can read more about reactive FormControls here: https://angular.io/guide/reactive-forms#getting-started in simple 3 steps.

Reply:
Later remove `FormBuilder` dependency from constructor as it won't be needed anymore (Let's simplify the code a bit:)